pkg/services/accesscontrol/roles.go

```diff
@@ -125,8 +131,9 @@ var usersAdminEditRole = RoleDTO{
 }
 
 var provisioningAdminRole = RoleDTO{
-	Name:    provisioningAdmin,
-	Version: 1,
+	Name:        provisioningAdmin,
+	Version:     2,
+	Description: "Allows to reload provisioning configurations.",
```

Comment:
```suggestion
	Description: "Allows to reload provisioning configurations.",
```
Allows [who? / what?] to reload ...

Reply:
This I know :smile:, this is mine
We have an endpoint that triggers the reloading (re-read, re-apply) of the provisioning configuration files.
I think the description is spot on.
If we really have to change, maybe:
```suggestion
	Description: "Role to allow users to reload the provisioning configuration files.",
```